src/freenet/client/async/SplitFileInserter.java

```diff
@@ -175,6 +175,7 @@ public void onFinishedEncode() {
 
     @Override
     public void encodingProgress() {
+        if(ctx.getCHKOnly) return;
```

Comment:
Why does making a change to `encodingProgress()` change what's done on encoding finish? That isn't `onFinishedEncode()`?

It wasn't immediately clear to me that `getCHKOnly` meant to only generate the key, which made this confusing. Makes sense now, but how about a comment? Maybe something like

    // This request is only to generate the key. Do not encode.

This line needs a space between `if` and `(` to meet the [coding standards](https://google-styleguide.googlecode.com/svn/trunk/javaguide.html#s4.6.2-horizontal-whitespace).